.github/workflows/interop.yml

```diff
@@ -35,6 +41,12 @@ jobs:
     name: Run hole-punching interoperability tests
     runs-on: ubuntu-22.04
     steps:
+      - name: Free Disk Space
+        # For some reason the original job (libp2p/test-plans) has enough disk space, but this one doesn't.
+        uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          tool-cache: true
+
```

Comment:
Is this currently failing?

Reply:
Yes, we are getting a not enough space error, as reported by Shashank on discord.

Reply:
If I understood correctly, only the transport test is failing.